### PR TITLE
add mpv.net support

### PR DESCRIPTION
**Pull request by brainless-afk**
Hi, I don't know if you care about supporting mpv.net (https://github.com/mpvnet-player/mpv.net), but with this the script would also work for it.

Also, the output argument --o="" is missing a minus, which doesn't seem to bother mpv, but is a problem for mpv.net.

### CODE REVIEW

**Comment by tatsumoto-ren** [comment]
No I haven't heard of mpv.net before.

File: encoder.lua [hunk 1]
```diff
@@ -21,10 +21,10 @@ local function construct_output_filename_noext()
...
 
     filename = string.format(
-            '%s_%s-%s',
-            filename,
-            h.human_readable_time(this.timings['start']),
-            h.human_readable_time(this.timings['end'])
+        '%s_%s-%s',
+        filename,
+        h.human_readable_time(this.timings['start']),
+        h.human_readable_time(this.timings['end'])
```
Comment (tatsumoto-ren) [inline encoder.lua:27]: There's no meaningful change here.

File: encoder.lua [hunk 2]
```diff
@@ -161,7 +161,7 @@ this.create_clip = function(clip_type, on_complete)
         if ret.status ~= 0 or string.match(ret.stdout, "could not open") then
             h.notify(string.format("Error: couldn't create clip %s.", output_file_path), "error", 5)
         else
-            h.notify(string.format("Clip saved to %s.", output_dir_path), "info", 2)
+            h.notify(string.format("Clip saved to %s.", output_file_path), "info", 2)
```
Comment (tatsumoto-ren) [inline encoder.lua:164]: What's the reason behind this change?

File: encoder.lua [hunk 3]
```diff
@@ -173,12 +173,24 @@ this.create_clip = function(clip_type, on_complete)
 end
 
 this.set_encoder_alive = function()
+    local args_mpvnet = { 'mpvnet', '--version' }
+    local process_result_mpvnet = function(_, ret, _)
+        --  for some reason stdout is empty
+        if ret.status ~= 0 then
+            this.alive = false
```
Comment (tatsumoto-ren) [inline encoder.lua:180]: `this.alive` isn't used in the code to test anything. I guess it could be used to panic if alive is false, but as of now you can remove it.

File: videoclip.lua [hunk 4]
```diff
@@ -331,9 +331,9 @@ pref_menu.litterbox_expirations = { '1h', '12h', '24h', '72h', }
 
 function pref_menu:get_selected_resolution()
     return string.format(
-            '%s x %s',
-            config.video_width == -2 and 'auto' or config.video_width,
-            config.video_height == -2 and 'auto' or config.video_height
+        '%s x %s',
+        config.video_width == -2 and 'auto' or config.video_width,
+        config.video_height == -2 and 'auto' or config.video_height
```
Comment (tatsumoto-ren) [inline videoclip.lua:336]: There's no meaningful change here.

File: videoclip.lua [hunk 5]
```diff
@@ -469,4 +469,3 @@ end
 validate_config()
 encoder.init(config, main_menu.timings)
 mp.add_key_binding('c', 'videoclip-menu-open', main_menu.open)
-mp.msg.warn("Press 'c' to open the videoclip menu.")
```
Comment (tatsumoto-ren) [inline videoclip.lua:472]: We need to keep this line to remind the user how to open the main menu.

**Review by tatsumoto-ren — COMMENTED** [review]
Left some thoughts.

File: encoder.lua (hunk 2 above)
Comment (brainless-afk) [reply encoder.lua:164]: I found it strange that the clip created message only contains the directory and not the name of the file created.

File: encoder.lua (hunk 1 above)
Comment (brainless-afk) [reply encoder.lua:27]: yeah, just formatting

File: encoder.lua (hunk 3 above)
Comment (brainless-afk) [reply encoder.lua:180]: encoder.alive is used in videoclip.lua line 259

File: videoclip.lua (hunk 5 above)
Comment (brainless-afk) [reply videoclip.lua:472]: yes, i get that, but if you adjust your input.config, the text is no longer correct. Likewise, if you have already set an input to c, then it will not be correct either and not open the menu
